packages/configs/default/index.json

```diff
@@ -32,6 +32,7 @@
     "script:*.vue": ["@parcel/transformer-vue"],
     "style:*.vue": ["@parcel/transformer-vue"],
     "custom:*.vue": ["@parcel/transformer-vue"],
+    "url:*.{png,jpg,jpeg,webp}": ["@parcel/transformer-image", "..."],
```

Comment:
hmm so now you couldn't use it from CSS/HTML without the `url:` prefix either... :/

Reply:
that's probably why I originally did it the other way...

Reply:
so should I change it back?